.github/dependabot.yml

```diff
@@ -12,3 +7,30 @@ updates:
       interval: "daily"
     open-pull-requests-limit: 5
     target-branch: "master"
+    groups:
+      jakarta:
+        patterns:
+        - "jakarta.*"
+        update-types:
+        - "patch"
+      microprofile:
+        patterns:
+        - "org.eclipse.microprofile*"
+        update-types:
+        - "patch"
```

Comment:
I don't think we want these grouped. I'd continue letting them being updated one at a time
```suggestion
```

Reply:
removed groupng

Reply:
I don't think you have?